package.json

```diff
@@ -18,6 +18,7 @@
   "devDependencies": {
     "@commitlint/cli": "17.0.2",
     "@commitlint/config-conventional": "17.0.2",
+    "@timbeyer/commitlint-circle": "^4.1.1",
```

Comment:
why this dep change?

Reply:
The previous dependency was behind and we were breaking due to conflicting dependencies in our tree. This one appears to be more regularly up-to-date, though I'm not positive if it changes the output in circle much.